app/assets/stylesheets/common/base/code_highlighting.scss

```diff
@@ -1,24 +1,25 @@
-/* originally based on github.com style (c) Vasily Polovnyov <vast@whiteants.net> */
-
 p > code,
 li > code,
-pre > code,
 strong > code,
 em > code {
-  font-family: ui-monospace, "Cascadia Mono", "Segoe UI Mono", "Liberation Mono",
-    Menlo, Monaco, Consolas, monospace;
+  padding: 2px 4px;
+  background: var(--inline-code-bg);
+  white-space: pre-wrap;
+  color: var(--primary);
+}
+
+code {
   color: var(--primary-very-high);
   background: var(--hljs-bg);
```

Comment:
So the `--hljs-bg` color applies now to all code blocks except for those right below `p`, `em`, `li`... it's a little confusing.

Reply:
I've added a screenshot. The large codeblocks will have their own background, and any inline codeblocks will have a *slightly* darker background. This is how stack overflow also renders their codeblocks. The bg color is different in linline versus a whole block.